SQRLPlatformAwareInstaller/ViewModels/VersionSelectorViewModel.cs

```diff
@@ -52,16 +53,18 @@ public VersionSelectorViewModel()
 
         private void InitObj(string platform ="WINDOWS")
         {
+            //Handle certificate trust Issue from Issue #80
+            ServicePointManager.ServerCertificateValidationCallback = (a, b, c, d) => true;
```

Comment:
If this is doing what I think it's doing, then we have a big security problem!
This enables easy DNS spoofing attacks, letting a MITM easily install fake "updates" from a spoofed site. I strongly suggest to remove this. Or am I missing something?

Reply:
Yeah I was trying to fix an issue someone ran into at the sqrl forums where they got an untrusted certificate issue 
You are right though I didn’t think it mattered that much since it’s just on the installer but I don’t like it overall just was trying to fix this one issue for this person but you are right I’ll remove it

Reply:
ok @alexhauser  I've fixed this and pushed it. Sorry about my short sight hehe